build.gradle

```diff
@@ -89,3 +93,7 @@ testClusters.integTest {
 run {
     useCluster testClusters.integTest
 }
+
+tasks.withType(RestIntegTestTask)*.configure {
+  classpath += files(project.configurations.runtimeClasspath.findAll { it.name.contains("log4j-core") })
+}
```

Comment:
Why is `log4j-core` special?

Reply:
@saratvemulapalli see please https://github.com/opensearch-project/opensearch-plugin-template-java/pull/16#issuecomment-1000536819, it is need but is optional

Reply:
Thanks @reta , I did read through the comments on https://github.com/dblock/opensearch-plugin-template-java/pull/1. 
I still dont understand why :)

Reply:
There is something coming from the plugin apparently, I will create an issue shortly to figure out exactly what is going on, will ping you there, thanks!